README.md

```diff
@@ -1,63 +1,107 @@
+Mautic is the world’s largest open source marketing automation project. With over 200,000 organisations using Mautic and over 1,000 community volunteers, we empower businesses by making it easy to manage their marketing across a range of channels. 
+
+### ✨ Benefits over regular marketing automation tools
+
+- 🌐 It's open-source - use it for free, no strings attached
+- 🏠 Don't rely on third-party SaaS, run your own marketing automation
+- 🗄️ Use your own database and own your data
+- 🔓 No vendor lock-in - switch or modify at will
+- 🔧 Create any integration you need
+- ⚡ Build stronger campaigns, without feature limitations
+- 🛠️ Both frontend and backend are 100% customizable
+- 📡 Multi-channel marketing without extra costs
+- 📊 Unlimited segmentation and automation
+
+Stay up to date about initiatives, releases and strategy via our [blog][mautic-blog].
 
 Marketing automation has historically been difficult to implement within organisations. The Mautic Community is an example of open source at its best, offering great software and a vibrant and caring community in which to learn and share knowledge.
 
 Open source means more than open code. Open source provides equality for all and a chance for everyone to improve.
 
+
 ![Mautic](.github/readme_image.png "Mautic Open Source Marketing Automation")
 
-Get Involved
+Get Involved Quickly
 =============
-Before we tell you how to install and use Mautic, we like to shamelessly plug our awesome user and developer communities! Users, start [here][get-involved] for inspiration, or follow us on Twitter [@MauticCommunity][twitter] or Facebook [@MauticCommunity][facebook]. Once you’re familiar with using the software, maybe you will share your wisdom with others in our [Slack][slack] channel.
+Calling all devs, testers and tech writers! Get started quickly and show us what you’re made of. Technical contributions are also welcome.
+
+- 🐛 [Easy-to-fix bugs][t1-issues]
+- 🧪 [Best pull requests for testing][best-pulls]
 
-Calling all devs, testers and tech writers! Technical contributions are also welcome. First, read our [general guidelines][contributing] about contributing. If you want to contribute code, read our [CONTRIBUTING.md][contributing-md] or [Contributing Code][contribute-developer] docs then check out the issues with the [T1 label][t1-issues] to get stuck in quickly and show us what you’re made of.
+If you have questions, [check how to contribute](#-contribution).
+
+Using Mautic
+============
 
-If you have questions, the Mautic Community can help provide the answers.
+### ☁️ Deploy instantly with our Trials
 
-Installing and using Mautic
-============================
+[Request a trial][mautic-trials] filling up a form to deploy in minutes and test Mautic now.
 
-## Supported Versions
+### 🚀 Self-host completely free, forever
 
 Please check the latest supported versions on the [Mautic Releases](https://www.mautic.org/mautic-releases) page.
```

Comment:
If we are using link macros elsewhere, let's also use them here?

Reply:
In this case yeah
There are some links styled differently that just don't work with the macros, but the macros don't work with HTML aligment
anyway, I'll fix where it's possible